src/components/Flex/Flex.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { mount, shallow } from 'enzyme';
+import React from 'react';
+
+import { Flex } from './Flex';
+
+describe('<Flex />', () => {
+	it('Should be able to render', () => {
+		shallow(<Flex />);
+	});
+
+	it('Should set the correct className', () => {
+		const flexComponent = shallow(<Flex />);
+
+		expect(flexComponent.hasClass('o-flex')).toEqual(true);
+	});
+
+	it('Should set the correct className when orientation is passed', () => {
+		const flexComponent = shallow(<Flex orientation="horizontal" />);
+
+		expect(flexComponent.hasClass('o-flex--horizontal')).toEqual(true);
+	});
+
+	it('Should set the correct className when center is passed', () => {
+		const flexComponent = shallow(<Flex center />);
+
+		expect(flexComponent.hasClass('o-flex--center')).toEqual(true);
+	});
+
+	it('Should set the correct className when orientation and center are passed', () => {
+		const flexComponent = shallow(<Flex orientation="horizontal" center />);
+
+		expect(flexComponent.hasClass('o-flex--horizontal-center')).toEqual(true);
+	});
+
+	it('Should set the correct className when justify is passed', () => {
+		const flexComponent = shallow(<Flex justify />);
+
+		expect(flexComponent.hasClass('o-flex--justify-between')).toEqual(true);
+	});
+
+	it('Should set the correct className when spaced is passed (boolean)', () => {
```

Comment:
add a test for "Should not add any spaced classes when no spaced option is passed"